Script/version.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+set -e
```

Comment:
It's best to store all the scripts in a single directory. We can move this script to the https://github.com/adobe/aepsdk-roku/tree/dev/build directory and change the name of the directory if needed.